content/300-guides/200-deployment/700-patching-production.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Patching / hotfixing
+metaDescription: How to reconcile the migration history after applying a hotfix or patch to a production environment.
+---
+
+<TopBlock>
+
+Patching or hotfixing a database involves making an often time critical change directly in production. For example, you might add an index directly to a production database to resolve an issue with a slow-running query.
+
+Patching the production database directly results in **schema drift**: your database schema has 'drifted away' from the source of truth, and is out of sync with your migration history. You can use the `prisma migrate resolve` command to reconcile your migration history _without_ having to remove and re-apply the hotfix with `prisma migrate deploy`.
+
+</TopBlock>
+
+## Reconciling your migration history with a patch or hotfix
+
+The following scenario assumes that you made a manual change in production and want to propagate that change to your migration history and other databases.
+
+To reconcile your migration history and database schema in production:
+
+1. Replicate the change you made in production in the schema - for example, add an `@@index` to a particular model.
+1. Generate a new migration:
+
+   ```terminal
+   npx prisma migrate dev --name retroactively-add-index
+   ```
+
+1. Push the migration to production **without running `migrate deploy`**. Instead, mark the migration as 'already applied' so that Prisma Migrate does not attempt to apply your hotfix a second time:
```

Comment:
```suggestion
1. Push the migration to production **without running `migrate deploy`**. Instead, mark the migration created in the previous step as 'already applied' so that Prisma Migrate does not attempt to apply your hotfix a second time:
```